drivers/setup/blue/blue.c

```diff
@@ -1073,16 +1075,28 @@ ScrIoControl(
 
             if (DeviceExtension->Enabled && DeviceExtension->VideoMemory)
             {
+#ifdef OPTIMIZED
+                CHAR attr = Buf->wAttribute;
```

Comment:
When you cache wAttr, use `UCHAR` variable instead.

Reply:
Fixed.